robot_skills/package.xml

```diff
@@ -41,6 +41,7 @@
 
   <test_depend>test_tools</test_depend>
   <test_depend>python-six</test_depend>
+  <test_depend>hero_description</test_depend>
```

Comment:
I think this is a very dirty dependency. But this might be fixed by @jlunenburg.

Reply:
It is dirty. But it is a dependency that we have always had implicitly as we cant construct a robot object without its description.